app/helpers/early_allocation_helper.rb

```diff
@@ -24,4 +72,11 @@ def active_status(early_allocation)
       'Not eligible'
     end
   end
+
+private
+
+  def get_most_recent(early_allocations)
```

Comment:
it is considered poor practice to perform queries in view helpers - surely we already have the data required, and in the correct order? - so early_allocations.last would work here?